dist/assets/style.css

```diff
@@ -0,0 +1,94 @@
+body {
+  height: 100vh;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.body {
+  background: -webkit-gradient(linear, left top, right top, from(rgba(243, 237, 237, 0.335)), to(rgba(128, 128, 128, 0.315))), url("./images/background.jpg");
+  background: linear-gradient(to right, rgba(243, 237, 237, 0.335), rgba(128, 128, 128, 0.315)), url("./images/background.jpg");
+}
+
+.searchbar {
+  text-align: center;
+  background-color: #e2dddd;
+  width: 55%;
+  margin: 0 auto;
+  border-radius: 10px;
+  padding: 10px 2px;
+}
+
+.searchbar label {
+  font-size: 15px;
+}
+
+.search_button {
+  color: #fdfdfd;
+  background-color: #424040;
+  border: 1px solid gray;
+  border-radius: 5px;
+  padding: 3px;
+  font-size: 17px;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  cursor: pointer;
+}
+
+.search_button:hover {
+  color: #424040;
+  background-color: #fdfdfd;
+}
+
+.container {
+  border: 2px solid white;
+  margin: 5px 10px;
+  height: 70vh;
+  background-color: rgba(255, 255, 255, 0.4);
+  text-align: center;
+  position: relative;
+}
+
+.container .result {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  -webkit-transform: translate(-50%, -50%);
+          transform: translate(-50%, -50%);
+}
+
+.error {
+  color: #c72424;
+}
+
+.rain {
```

Comment:
- [ ] [OPTIONAL] All the weather bgs are repeating on desktop width. Make sure to add a background-size property to make your project nicer to look at.
![Screenshot_2020-09-11 Weather(1)](https://user-images.githubusercontent.com/16193212/92930520-8ff7de00-f418-11ea-99fa-0c4e5873b532.png)